src/Microsoft.VisualStudio.SlnGen/ProgramArguments.cs

```diff
@@ -85,6 +85,16 @@ public sealed class ProgramArguments
             ShowInHelpText = false)]
         public bool Debug { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not to disable building projects for configurations that are not supported by those projects
+        /// </summary>
+        [Option(
+            "--disabledefaultconfig",
+            CommandOptionType.MultipleValue,
+            ValueName = "true",
+            Description = "Disable building projects for configurations that are not supported by those projects.  Default: false")]
+        public string[] DisableDefaultConfig { get; set; }
```

Comment:
Let's name this `AlwaysBuild` and have it default to `true` since this is the current default behavior.  Then if you want it off, you have to set it to `false`.  The description can say something like "Always include the project in the build even if it has no matching configuration"